template/flexLayout.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
```

Comment:
please write a comment explaining this file

Reply:
added. Let me know if it needs to be more in-depth, wasn't sure how much it needed to explain.